### PR TITLE
prometheus operator and config-reloader: new package

### PR DESCRIPTION
**Pull request by found-it**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

#### For security-related PRs

- [ ] The security fix is recorded in `annotations` and `secfixes`

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

File: prometheus-config-reloader.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: prometheus-config-reloader
+  version: 0.64.0
+  epoch: 0
+  description: Prometheus Operator creates/configures/manages Prometheus clusters atop Kubernetes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime: []
```
Comment (imjasonh) [inline prometheus-config-reloader.yaml:9]:
```suggestion
```

File: prometheus-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: prometheus-operator
+  version: 0.64.0
+  epoch: 0
+  description: Prometheus Operator creates/configures/manages Prometheus clusters atop Kubernetes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime: []
```
Comment (imjasonh) [inline prometheus-operator.yaml:9]:
```suggestion
```

File: prometheus-operator.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    with:
+      repository: https://github.com/prometheus-operator/prometheus-operator.git
+      tag: v${{package.version}}
+      destination: ${{package.name}}
+  - working-directory: ${{package.name}}
+    pipeline:
+      - uses: autoconf/make
+        with:
+          opts: |
+            operator
```
Comment (imjasonh) [inline prometheus-operator.yaml:30]:
```suggestion
          opts: operator
```

**Comment by found-it** [comment]
Just a heads up, I added `make test-unit` here as an extra check. Any objections?

**Review by imjasonh — APPROVED** [review]

**Comment by imjasonh** [comment]
> Just a heads up, I added `make test-unit` here as an extra check. Any objections?

This seems fine for now. If we add an explicit test scenario section we'll want to move this, but I don't mind as long as it's not too slow.

**Pull request merged**